corona-warn-app/cwa-app-android
# Update expired certificate details screen with new status subtitle.

Small text change for the expired certificate details subtitle.

To test, generate an expiring certificate:

cwa vc gen --exp 5minutes -e mock-cloud -> go to certificate details and check the subtitle below the QR code. Compare with figma: https://www.figma.com/file/KuDEcMSREg3VhZRhC3FXTY/COVID19_Master_2.20?node-id=50171%3A36132
